.gitlab-ci.yml

```diff
@@ -131,24 +131,25 @@ release-prod-tag:
   <<: *docker-hub-login
   stage: release-public
   tags: ["runner:docker"]
-  image: registry.ddbuild.io/docker-notary:0.6.1
```

Comment:
We need to use the docker-push image, this docker-notary can't use `buildx build`

Reply:
what was docker-notary used for? 😅

Reply:
it was a smaller container that couldn't build, but did have the right setup to pull/tag/push